### DISCUSSION TITLE
Tau on GoNix

### DISCUSSION THREAD

**Discussion by prologic** [NicoNex/tau#34] [Show and tell]
Here's `tau` as an applet on [GoNix](https://git.mills.io/prologic/gonix) 🥳

<img width="1196" alt="Screen Shot 2022-08-03 at 09 45 21" src="https://user-images.githubusercontent.com/1290234/182495395-bb02478e-8f02-426d-9bf1-1b70bb1a37b9.png">

**Comment by NicoNex** [comment]
Awesome ❤️
